src/libical/icaltime.c

```diff
@@ -145,12 +145,17 @@ static time_t make_time(struct tm *tm, int tzm)
 
     tim = (time_t) ((tm->tm_year - 70) * 365 + ((tm->tm_year - 1) / 4) - 17);
 
+    /* adjust: no leap days every 100 years, except every 400 years. */
+
+    int febs = (tm->tm_year - 100) - ((tm->tm_mon <= 1) ? 1 : 0);
```

Comment:
need to declare 'int febs' near the top of the function, to support ancient C

Reply:
Changed as suggested. Although this should be supported by C99, which is the minimum we require.

Reply:
you're right.  I'm using a tool (splint) to check for ancient C support.  I'll see if I can tell splint to allow C99